en/admin/config.rst

```diff
@@ -1126,6 +1134,13 @@ The following are parameters related to logs used for database backup and restor
 
     **sync_on_flush_size** is a parameter to configure the interval in pages between after data and log pages are flushed from buffer and before they are synchronized with FILE I/O of operating system. The default value is 200 * :ref:`db_page_size <dpg>` (**3.125M** when db_page_size is 16K). That is, the CUBRID Server performs synchronization with the FILE I/O of the operating system whenever 200 pages have been flushed. This is also a parameter related to I/O load.
 
+**ddl_audit_log**
+	**ddl_audit_log** is a parameter to configure whether to log. The default value is no.  
+	If this value is yes, it is saved in the log file when DDL SQL is executed. The path to save the log file is log/ddl_audit, and refer to :doc:`/admin/ddl_audit` for the DDL AUDIT log file name.
```

Comment:
If this value is set to yes, all DDL executed will be logged into the logfile. The path of log files is $CUBRID/log/ddl_audit, and refer to :doc:`/admin/ddl_audit` for each DDL AUDIT log file name and description of log files in detail.

Reply:
Thank yoy, I'll change the sentence.